### ISSUE TITLE
Advanced sclerotic lupus nephritis

### ISSUE THREAD

**Issue by RichardKlaus** [obophenotype/human-phenotype-ontology#6837]
**Preferred term label:**
Advanced sclerotic lupus nephritis

**Synonyms**
Class VI lupus nephritis

**Definition (free text, please give PubMed ID)**
A subclass of lupus nephritis with ≥90% of glomeruli globally sclerosed without residual activity.
PMID: 31974366

**Parent term (use hpo.jax.org/app)**
Lupus nephritis (newly proposed term)

**Related terms**
Glomerular sclerosis HP:0000096
Systemic lupus erythematosus HP:0002725

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM #152700, ORPHA:536

**Your nano-attribution (ORCID)**
0000-0002-2234-4248

**Comment by pnrobinson** [comment]
added
